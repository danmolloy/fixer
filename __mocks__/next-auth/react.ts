
export const useSession = jest.fn()

useSession.mockImplementation(() => {
  const originalModule = jest.requireActual('next-auth/react');
  const mockSession = {
    userData: {
      playerCalls: []
    }
  }
  return {
    __esModule: true,
    ...originalModule,
    useSession: jest.fn(() => {
      return {data: mockSession, status: 'authenticated'}  // return type is [] in v3 but changed to {} in v4
    }),
  };
});