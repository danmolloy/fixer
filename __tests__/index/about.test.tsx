import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom"
import About from "../../components/index/about";
import React from "react";

describe("About component", () => {
  beforeEach(() => {
    render(<About />)
  })
  it("Renders", () => {
    const aboutDiv = screen.getByTestId("about-div")
    expect(aboutDiv).toBeInTheDocument()
  })
  it("Matches snapshot", () => {
    const aboutDiv = screen.getByTestId("about-div")
    expect(aboutDiv).toMatchSnapshot()
  })
})