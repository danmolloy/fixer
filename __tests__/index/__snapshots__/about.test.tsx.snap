// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component Matches snapshot 1`] = `
<div
  class="w-screen "
  data-testid="about-div"
>
  <div
    class="p-6 mt-12 pt-12"
  >
    <h1
      class="text-4xl "
    >
      How it works
    </h1>
  </div>
  <div>
    <div
      class="border bg-white text-black min-h-[75vh] mx-2 my-8 py-4 rounded-lg flex flex-col items-center"
    >
      <div
        class=" w-full flex flex-col px-6"
      >
        <h2
          class="text-3xl  py-2 "
        >
          Player Directory
        </h2>
      </div>
      <div
        class="p-4 text-zinc-600"
      >
        <p>
          Find players in our modern directory.
        </p>
      </div>
      <div
        class=" m-2  p-2 flex flex-col items-center w-[95%] sm:w-[75%] lg:w-1/2"
      >
        <div
          class="w-screen flex flex-col items-center"
          data-testid="player-directory-div"
          id="player-directory"
        >
          <div
            class="flex flex-col w-full items-center"
          >
            <select
              class="border shadow-sm p-1 rounded w-1/2 sm:w-1/3"
            >
              <option
                value=""
              >
                Select instrument
              </option>
              <option
                value="Viola"
              >
                Viola
              </option>
            </select>
          </div>
          <div>
            <h3
              class="p-16 text-slate-700"
            >
              Please select an instrument.
            </h3>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div>
    <div
      class="border bg-white text-black min-h-[75vh] mx-2 my-8 py-4 rounded-lg flex flex-col items-center"
    >
      <div
        class=" w-full flex flex-col px-6"
      >
        <h2
          class="text-3xl  py-2 "
        >
          Gig Interface
        </h2>
      </div>
      <div
        class="p-4 text-zinc-600"
      >
        <p>
          Create an event with our intuitive form. All players and admin view the same information, so no one misses any details.
        </p>
      </div>
      <div
        class=" m-2  p-2 flex flex-col items-center w-[95%] sm:w-[75%] lg:w-1/2"
      >
        <div
          class="w-full lg:w-2/3 flex flex-col items-center"
          data-testid="event-index-div"
        >
          <div
            class="w-full border shadow rounded-lg py-4"
            data-testid="event-info-div"
          >
            <div
              class="w-full flex flex-col"
            >
              <button
                class="self-end border p-1 mr-2 rounded-md shadow-sm border-amber-600 text-amber-600 bg-white hover:bg-amber-50"
                data-testid="options-btn"
              >
                Options
              </button>
            </div>
            <div
              class=""
            >
              <p
                class="flex flex-row items-center justify-center text-center p-3"
              >
                This event is 
                confirmed
              </p>
            </div>
            <div
              class="bg-slate-50 flex flex-col lg:flex-row p-4 w-full lg:items-center lg:justify-evenly "
              data-testid="ensemble-name"
            >
              <p
                class="text-slate-600 text-sm lg:w-1/2"
              >
                Ensemble
              </p>
              <p
                class="lg:w-1/2"
              >
                London Symphony Orchestra
              </p>
            </div>
            <div
              class="flex flex-col lg:flex-row p-4 w-full  lg:items-center lg:justify-evenly"
              data-testid="event-calls-list"
            >
              <p
                class="text-slate-600 text-sm lg:w-1/2 "
                data-testid="event-calls-count"
              >
                2
                 Call(s):
              </p>
              <div
                class="flex flex-col lg:w-1/2 "
              >
                <div
                  class="my-2"
                >
                  <div
                    class=""
                    data-testid="call-tile-div"
                  >
                    <p>
                      10:30 17 July 2023
                       
                      <span
                        class="text-sm"
                      >
                        to
                      </span>
                    </p>
                    <p>
                      16:33 31 July 2023
                    </p>
                    <div
                      class="text-slate-600 flex flex-row items-center"
                    >
                      <svg
                        fill="currentColor"
                        height="1em"
                        stroke="currentColor"
                        stroke-width="0"
                        viewBox="0 0 20 20"
                        width="1em"
                        xmlns="http://www.w3.org/2000/svg"
                      >
                        <path
                          clip-rule="evenodd"
                          d="M5.05 4.05a7 7 0 119.9 9.9L10 18.9l-4.95-4.95a7 7 0 010-9.9zM10 11a2 2 0 100-4 2 2 0 000 4z"
                          fill-rule="evenodd"
                        />
                      </svg>
                      <p
                        class="ml-2"
                      >
                        St Lukes
                      </p>
                    </div>
                  </div>
                </div>
                <div
                  class="my-2"
                >
                  <div
                    class=""
                    data-testid="call-tile-div"
                  >
                    <p>
                      16:33 31 July 2023
                       
                      <span
                        class="text-sm"
                      >
                        to
                      </span>
                    </p>
                    <p>
                      16:33 31 July 2023
                    </p>
                    <div
                      class="text-slate-600 flex flex-row items-center"
                    >
                      <svg
                        fill="currentColor"
                        height="1em"
                        stroke="currentColor"
                        stroke-width="0"
                        viewBox="0 0 20 20"
                        width="1em"
                        xmlns="http://www.w3.org/2000/svg"
                      >
                        <path
                          clip-rule="evenodd"
                          d="M5.05 4.05a7 7 0 119.9 9.9L10 18.9l-4.95-4.95a7 7 0 010-9.9zM10 11a2 2 0 100-4 2 2 0 000 4z"
                          fill-rule="evenodd"
                        />
                      </svg>
                      <p
                        class="ml-2"
                      >
                        Barbican Hall
                      </p>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="bg-slate-50 flex flex-col lg:flex-row p-4 w-full lg:items-center lg:justify-evenly "
              data-testid="event-program"
            >
              <p
                class="text-slate-600 text-sm lg:w-1/2"
              >
                Program
              </p>
              <p
                class="lg:w-1/2"
              >
                Sibelius Symphony 2
              </p>
            </div>
            <div
              class=" flex flex-col lg:flex-row p-4 w-full lg:items-center lg:justify-evenly "
              data-testid="event-dress"
            >
              <p
                class="text-slate-600 text-sm lg:w-1/2"
              >
                Dress
              </p>
              <p
                class="lg:w-1/2"
              >
                Tails/Long Black
              </p>
            </div>
            <div
              class="bg-slate-50 flex flex-col lg:flex-row p-4 w-full lg:items-center lg:justify-evenly "
              data-testid="event-fee"
            >
              <p
                class="text-slate-600 text-sm lg:w-1/2"
              >
                Fee
              </p>
              <p
                class="lg:w-1/2"
              >
                MU/ABO 1
              </p>
            </div>
            <div
              class=" flex flex-col lg:flex-row p-4 w-full lg:items-center lg:justify-evenly "
              data-testid="event-additional-info"
            >
              <p
                class="text-slate-600 text-sm lg:w-1/2"
              >
                Additional Info
              </p>
              <p
                class="lg:w-1/2"
              >
                Bring a stand
              </p>
            </div>
            <div
              class="bg-slate-50 flex flex-col lg:flex-row p-4 w-full lg:items-center lg:justify-evenly "
              data-testid="event-fixer-name"
            >
              <p
                class="text-slate-600 text-sm lg:w-1/2"
              >
                Fixer
              </p>
              <p
                class="lg:w-1/2"
              >
                Roy Dereks
              </p>
            </div>
            <div
              class="text-slate-600 text-sm flex flex-col lg:flex-row p-4 w-full lg:items-center lg:justify-evenly "
              data-testid="created-datetime"
            >
              <p
                class="text-slate-600 text-sm lg:w-1/2"
              >
                Event created
              </p>
              <p
                class="lg:w-1/2"
              >
                16:33:59 31 July 2023
              </p>
            </div>
            <div
              class="text-slate-600 text-sm bg-slate-50 flex flex-col lg:flex-row p-4 w-full lg:items-center lg:justify-evenly "
              data-testid="updated-datetime"
            >
              <p
                class="text-slate-600 text-sm lg:w-1/2"
              >
                Last updated
              </p>
              <p
                class="lg:w-1/2"
              >
                16:33:59 31 July 2023
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div>
    <div
      class="border bg-white text-black min-h-[75vh] mx-2 my-8 py-4 rounded-lg flex flex-col items-center"
    >
      <div
        class=" w-full flex flex-col px-6"
      >
        <h2
          class="text-3xl  py-2 "
        >
          Automated Fixing
        </h2>
      </div>
      <div
        class="p-4 text-zinc-600"
      >
        <p>
          We handle the bookings and availability checks with our flexible fixing features.
        </p>
      </div>
      <div
        class=" m-2  p-2 flex flex-col items-center w-[95%] sm:w-[75%] lg:w-1/2"
      >
        <div
          class="w-full h-full"
          data-testid="Viola-instrument-tile"
        >
          <div
            class=" p-2"
            data-testid="tile-header-div"
          >
            <div
              class=" flex flex-row items-center justify-between"
            >
              <div
                class="flex flex-col "
              >
                <h2
                  class=""
                >
                  Viola
                </h2>
                <div
                  class="flex flex-col"
                >
                  <div
                    class="flex flex-row items-center"
                  >
                    <p
                      class="text-sm"
                    >
                      Booked 
                      1
                       player(s)
                    </p>
                    <div
                      class="text-green-600"
                    >
                      <svg
                        baseProfile="tiny"
                        fill="currentColor"
                        height="1em"
                        stroke="currentColor"
                        stroke-width="0"
                        version="1.2"
                        viewBox="0 0 24 24"
                        width="1em"
                        xmlns="http://www.w3.org/2000/svg"
                      >
                        <path
                          d="M16.972 6.251c-.967-.538-2.185-.188-2.72.777l-3.713 6.682-2.125-2.125c-.781-.781-2.047-.781-2.828 0-.781.781-.781 2.047 0 2.828l4 4c.378.379.888.587 1.414.587l.277-.02c.621-.087 1.166-.46 1.471-1.009l5-9c.537-.966.189-2.183-.776-2.72z"
                        />
                      </svg>
                    </div>
                  </div>
                </div>
                <div
                  class=" text-sm font-bold py-2 text-red-500"
                >
                  <p
                    data-testid="mock-note"
                  />
                </div>
              </div>
            </div>
          </div>
          <div
            class=" w-full flex flex-row justify-evenly "
            data-testid="tile-tab-bar"
          >
            <button
              class="py-2 border-b-2 border-blue-600 text-blue-600"
              data-testid="booking-tab-toggle"
            >
              Booking
            </button>
            <button
              class="text-slate-400 py-2"
              data-testid="availability-tab-toggle"
            >
              Availability
            </button>
          </div>
          <div
            class=""
            data-testid="booking-tab"
          >
            <div
              class="active-calls-div"
              data-testid="active-calls-div"
            >
              <div
                class="flex flex-col mx-2"
                data-testid="booking-table-div"
              >
                <div
                  class="MuiTableContainer-root css-rorn0c-MuiTableContainer-root"
                >
                  <table
                    class="MuiTable-root css-rqglhn-MuiTable-root"
                  >
                    <thead
                      class="MuiTableHead-root css-15wwp11-MuiTableHead-root"
                    >
                      <tr
                        class="MuiTableRow-root MuiTableRow-head css-1q1u3t4-MuiTableRow-root"
                      >
                        <th
                          class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-1ygcj2i-MuiTableCell-root"
                          scope="col"
                        >
                          Name
                        </th>
                        <th
                          class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-1ygcj2i-MuiTableCell-root"
                          scope="col"
                          title="17 July"
                        >
                          17/07
                        </th>
                        <th
                          class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-1ygcj2i-MuiTableCell-root"
                          scope="col"
                          title="31 July"
                        >
                          31/07
                        </th>
                        <th
                          class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-1ygcj2i-MuiTableCell-root"
                          scope="col"
                        />
                      </tr>
                    </thead>
                    <tbody
                      class="MuiTableBody-root css-apqrd9-MuiTableBody-root"
                    >
                      <tr
                        class="MuiTableRow-root css-1q1u3t4-MuiTableRow-root"
                      >
                        <td
                          class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
                        >
                          <p
                            class="text-zinc-300 "
                          >
                            Carlos Beatrice
                          </p>
                        </td>
                        <td
                          class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
                        >
                          <div
                            class="text-slate-400"
                          >
                            <svg
                              baseProfile="tiny"
                              fill="currentColor"
                              height="1em"
                              stroke="currentColor"
                              stroke-width="0"
                              version="1.2"
                              viewBox="0 0 24 24"
                              width="1em"
                              xmlns="http://www.w3.org/2000/svg"
                            >
                              <path
                                d="M17.414 6.586c-.78-.781-2.048-.781-2.828 0l-2.586 2.586-2.586-2.586c-.78-.781-2.048-.781-2.828 0-.781.781-.781 2.047 0 2.828l2.585 2.586-2.585 2.586c-.781.781-.781 2.047 0 2.828.39.391.902.586 1.414.586s1.024-.195 1.414-.586l2.586-2.586 2.586 2.586c.39.391.902.586 1.414.586s1.024-.195 1.414-.586c.781-.781.781-2.047 0-2.828l-2.585-2.586 2.585-2.586c.781-.781.781-2.047 0-2.828z"
                              />
                            </svg>
                          </div>
                        </td>
                        <td
                          class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
                        >
                          <div
                            class="text-slate-400"
                          >
                            <svg
                              baseProfile="tiny"
                              fill="currentColor"
                              height="1em"
                              stroke="currentColor"
                              stroke-width="0"
                              version="1.2"
                              viewBox="0 0 24 24"
                              width="1em"
                              xmlns="http://www.w3.org/2000/svg"
                            >
                              <path
                                d="M17.414 6.586c-.78-.781-2.048-.781-2.828 0l-2.586 2.586-2.586-2.586c-.78-.781-2.048-.781-2.828 0-.781.781-.781 2.047 0 2.828l2.585 2.586-2.585 2.586c-.781.781-.781 2.047 0 2.828.39.391.902.586 1.414.586s1.024-.195 1.414-.586l2.586-2.586 2.586 2.586c.39.391.902.586 1.414.586s1.024-.195 1.414-.586c.781-.781.781-2.047 0-2.828l-2.585-2.586 2.585-2.586c.781-.781.781-2.047 0-2.828z"
                              />
                            </svg>
                          </div>
                        </td>
                        <td
                          class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
                        >
                          <button
                            class="rounded-full p-1 text-zinc-700 hover:text-blue-600 hover:bg-blue-100"
                          >
                            <svg
                              fill="currentColor"
                              height="1em"
                              stroke="currentColor"
                              stroke-width="0"
                              viewBox="0 0 16 16"
                              width="1em"
                              xmlns="http://www.w3.org/2000/svg"
                            >
                              <path
                                d="M3 9.5a1.5 1.5 0 1 1 0-3 1.5 1.5 0 0 1 0 3zm5 0a1.5 1.5 0 1 1 0-3 1.5 1.5 0 0 1 0 3zm5 0a1.5 1.5 0 1 1 0-3 1.5 1.5 0 0 1 0 3z"
                              />
                            </svg>
                          </button>
                        </td>
                      </tr>
                      <tr
                        class="MuiTableRow-root css-1q1u3t4-MuiTableRow-root"
                      >
                        <td
                          class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
                        >
                          <p
                            class=""
                          >
                            Frank Zappa
                          </p>
                        </td>
                        <td
                          class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
                        >
                          <div
                            class="text-green-600"
                          >
                            <svg
                              baseProfile="tiny"
                              fill="currentColor"
                              height="1em"
                              stroke="currentColor"
                              stroke-width="0"
                              version="1.2"
                              viewBox="0 0 24 24"
                              width="1em"
                              xmlns="http://www.w3.org/2000/svg"
                            >
                              <path
                                d="M16.972 6.251c-.967-.538-2.185-.188-2.72.777l-3.713 6.682-2.125-2.125c-.781-.781-2.047-.781-2.828 0-.781.781-.781 2.047 0 2.828l4 4c.378.379.888.587 1.414.587l.277-.02c.621-.087 1.166-.46 1.471-1.009l5-9c.537-.966.189-2.183-.776-2.72z"
                              />
                            </svg>
                          </div>
                        </td>
                        <td
                          class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
                        >
                          <div
                            class="text-green-600"
                          >
                            <svg
                              baseProfile="tiny"
                              fill="currentColor"
                              height="1em"
                              stroke="currentColor"
                              stroke-width="0"
                              version="1.2"
                              viewBox="0 0 24 24"
                              width="1em"
                              xmlns="http://www.w3.org/2000/svg"
                            >
                              <path
                                d="M16.972 6.251c-.967-.538-2.185-.188-2.72.777l-3.713 6.682-2.125-2.125c-.781-.781-2.047-.781-2.828 0-.781.781-.781 2.047 0 2.828l4 4c.378.379.888.587 1.414.587l.277-.02c.621-.087 1.166-.46 1.471-1.009l5-9c.537-.966.189-2.183-.776-2.72z"
                              />
                            </svg>
                          </div>
                        </td>
                        <td
                          class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
                        >
                          <button
                            class="rounded-full p-1 text-zinc-700 hover:text-blue-600 hover:bg-blue-100"
                          >
                            <svg
                              fill="currentColor"
                              height="1em"
                              stroke="currentColor"
                              stroke-width="0"
                              viewBox="0 0 16 16"
                              width="1em"
                              xmlns="http://www.w3.org/2000/svg"
                            >
                              <path
                                d="M3 9.5a1.5 1.5 0 1 1 0-3 1.5 1.5 0 0 1 0 3zm5 0a1.5 1.5 0 1 1 0-3 1.5 1.5 0 0 1 0 3zm5 0a1.5 1.5 0 1 1 0-3 1.5 1.5 0 0 1 0 3z"
                              />
                            </svg>
                          </button>
                        </td>
                      </tr>
                    </tbody>
                  </table>
                </div>
              </div>
            </div>
            <div
              class="w-full flex flex-row justify-end"
            >
              <button
                class=" m-4 px-4 text-blue-500 border-blue-500 hover:bg-blue-100 border py-1 px-2 m-1 rounded shadow-sm flex items-center justify-center"
                data-testid="booking-edit-btn"
              >
                Edit
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div>
    <div
      class="border bg-white text-black min-h-[75vh] mx-2 my-8 py-4 rounded-lg flex flex-col items-center"
    >
      <div
        class=" w-full flex flex-col px-6"
      >
        <h2
          class="text-3xl  py-2 "
        >
          Personal Calendar
        </h2>
      </div>
      <div
        class="p-4 text-zinc-600"
      >
        <p>
          Once accepted, the dates are added to your player's calendars.
        </p>
      </div>
      <div
        class=" m-2  p-2 flex flex-col items-center w-[95%] sm:w-[75%] lg:w-1/2"
      >
        <div
          class="w-full border my-2 px-2 py-3 shadow-sm"
        >
          <h3
            class="text-md text-slate-700"
          >
            Monday 17th July 2023
          </h3>
          <div>
            <div
              class="flex flex-col items-center w-full"
            >
              <div
                class="w-full flex flex-row justify-end -mb-10"
              >
                <button
                  class="z-10 self-end text-xs mr-1 hover:bg-slate-100  rounded-full p-2 text-slate-800"
                  data-testid="event-menu-icon"
                >
                  •••
                </button>
              </div>
              <a
                class="rounded w-full border-b p-2 flex flex-col my-2 hover:bg-slate-100"
                data-testid="event-tile-div"
                href="/"
              >
                <div
                  class="flex flex-row justify-between"
                >
                  <h2
                    class=" text-lg"
                  >
                    Finnish Fantasy
                  </h2>
                </div>
                <p
                  class=""
                >
                  London Symphony Orchestra
                </p>
                <div
                  class="flex flex-col md:flex-row text-slate-800 md:py-2 md:mb-1"
                >
                  <div
                    class="flex flex-row items-center md:pr-2 md:border-r border-slate-400 "
                  >
                    <svg
                      fill="currentColor"
                      height="1em"
                      stroke="currentColor"
                      stroke-width="0"
                      viewBox="0 0 1024 1024"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M880 184H712v-64c0-4.4-3.6-8-8-8h-56c-4.4 0-8 3.6-8 8v64H384v-64c0-4.4-3.6-8-8-8h-56c-4.4 0-8 3.6-8 8v64H144c-17.7 0-32 14.3-32 32v664c0 17.7 14.3 32 32 32h736c17.7 0 32-14.3 32-32V216c0-17.7-14.3-32-32-32zm-40 656H184V460h656v380zM184 392V256h128v48c0 4.4 3.6 8 8 8h56c4.4 0 8-3.6 8-8v-48h256v48c0 4.4 3.6 8 8 8h56c4.4 0 8-3.6 8-8v-48h128v136H184z"
                      />
                    </svg>
                    <p
                      class="ml-2"
                      data-testid="call-start-time"
                    >
                      10:30am Mon 17th July 2023
                    </p>
                  </div>
                  <div
                    class="flex flex-row items-center md:pl-2 "
                  >
                    <svg
                      fill="currentColor"
                      height="1em"
                      stroke="currentColor"
                      stroke-width="0"
                      viewBox="0 0 20 20"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        clip-rule="evenodd"
                        d="M5.05 4.05a7 7 0 119.9 9.9L10 18.9l-4.95-4.95a7 7 0 010-9.9zM10 11a2 2 0 100-4 2 2 0 000 4z"
                        fill-rule="evenodd"
                      />
                    </svg>
                    <p
                      class="ml-2"
                    >
                      St Lukes
                    </p>
                  </div>
                </div>
              </a>
            </div>
          </div>
          <div>
            <div
              class="flex flex-col items-center w-full"
            >
              <div
                class="w-full flex flex-row justify-end -mb-10"
              >
                <button
                  class="z-10 self-end text-xs mr-1 hover:bg-slate-100  rounded-full p-2 text-slate-800"
                  data-testid="event-menu-icon"
                >
                  •••
                </button>
              </div>
              <a
                class="rounded w-full border-b p-2 flex flex-col my-2 hover:bg-slate-100"
                data-testid="event-tile-div"
                href="/"
              >
                <div
                  class="flex flex-row justify-between"
                >
                  <h2
                    class=" text-lg"
                  >
                    Finnish Fantasy
                  </h2>
                </div>
                <p
                  class=""
                >
                  London Symphony Orchestra
                </p>
                <div
                  class="flex flex-col md:flex-row text-slate-800 md:py-2 md:mb-1"
                >
                  <div
                    class="flex flex-row items-center md:pr-2 md:border-r border-slate-400 "
                  >
                    <svg
                      fill="currentColor"
                      height="1em"
                      stroke="currentColor"
                      stroke-width="0"
                      viewBox="0 0 1024 1024"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M880 184H712v-64c0-4.4-3.6-8-8-8h-56c-4.4 0-8 3.6-8 8v64H384v-64c0-4.4-3.6-8-8-8h-56c-4.4 0-8 3.6-8 8v64H144c-17.7 0-32 14.3-32 32v664c0 17.7 14.3 32 32 32h736c17.7 0 32-14.3 32-32V216c0-17.7-14.3-32-32-32zm-40 656H184V460h656v380zM184 392V256h128v48c0 4.4 3.6 8 8 8h56c4.4 0 8-3.6 8-8v-48h256v48c0 4.4 3.6 8 8 8h56c4.4 0 8-3.6 8-8v-48h128v136H184z"
                      />
                    </svg>
                    <p
                      class="ml-2"
                      data-testid="call-start-time"
                    >
                      4:33pm Mon 31st July 2023
                    </p>
                  </div>
                  <div
                    class="flex flex-row items-center md:pl-2 "
                  >
                    <svg
                      fill="currentColor"
                      height="1em"
                      stroke="currentColor"
                      stroke-width="0"
                      viewBox="0 0 20 20"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        clip-rule="evenodd"
                        d="M5.05 4.05a7 7 0 119.9 9.9L10 18.9l-4.95-4.95a7 7 0 010-9.9zM10 11a2 2 0 100-4 2 2 0 000 4z"
                        fill-rule="evenodd"
                      />
                    </svg>
                    <p
                      class="ml-2"
                    >
                      Barbican Hall
                    </p>
                  </div>
                </div>
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
